include/js/FieldDependencies.js

```diff
@@ -550,6 +550,15 @@ FieldDependencies.prototype.fieldReadonlyDetailView = function (readonlyFields)
 		document.getElementById('dtlview_'+field).style.display='inline';
 		document.getElementsByName(field).item(0).setAttribute('readonly', true);
 		document.getElementsByName(field).item(0).style.display='inline';
+
+		let elem = document.getElementById('txtbox_'+field);
+		var siblings = [];
+		while (elem = elem.nextElementSibling) {
```

Comment:
Why two loops?  Can't you just directly set the display in the first loop? as far as I see you don't use that siblings array for anything else.

Don't you have to "undo" this?  I mean, this is when a set of fields have to be hidden on a dependency, I understand that if that dependency value changes you would have to set these fields back to display=block, no?

Reply:
Changed to this
```js
let elem = document.getElementById('txtbox_'+field);
	while (elem = elem.nextElementSibling) {
	elem.style.display = 'none';
}
```
This for hide `save` and `cancel` buttons when fields are `readonly` on dependency

Reply:
and trying to edit on that field